heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraScheduler.java

```diff
@@ -190,6 +194,22 @@ protected String getHeronAuroraPath() {
     return auroraProperties;
   }
 
+  /**
+   * This method finds the container with highest resource requirement and returns the resource.
+   * Currently only RAM is used for max identification.
+   */
+  private Resource getMaxRequiredResource(PackingPlan packingPlan) {
```

Comment:
Could you file an issue regarding the cpu and disk update so that we can track it?